### PR TITLE
GUACAMOLE-338: Expand connections/groups by default if any of their descendants are selected.

### PR DESCRIPTION
**Pull request by mike-jumper**

### CODE REVIEW

File: guacamole/src/main/webapp/app/groupList/types/GroupListItem.js [hunk 1]
```diff
@@ -187,7 +187,7 @@ angular.module('groupList').factory('GroupListItem', ['ConnectionGroup', functio
             dataSource : dataSource,
 
             // Type information
-            expandable : includeSharingProfiles,
+            expandable : includeSharingProfiles !== false,
```
Comment (jmuehlner) [inline guacamole/src/main/webapp/app/groupList/types/GroupListItem.js]: Why this change?

**Comment by jmuehlner** [comment]
This looks like a good idea to me!

File: guacamole/src/main/webapp/app/groupList/types/GroupListItem.js (hunk 1 above)
Comment (mike-jumper) [inline guacamole/src/main/webapp/app/groupList/types/GroupListItem.js]: Sharing profiles are supposed to be included by default (the function is documented as `includeSharingProfiles` defaulting to `true`), but not specifying that parameter makes `includeSharingProfiles` falsey. The `expandable` property needs to be `true` unless `includeSharingProfiles` is explicitly `false`.

**Comment by necouchman** [comment]
Looks good to me...I'll get these commits into the master branch shortly.
